### PR TITLE
Adding new styles, fixes #47

### LINKED ISSUE #47: Add support for new print styles
Add support for the following styles in print CSS:

* ScreenplayHead sph
* ScreenplaySubhead spsh
* ScreenplayDramatisPersonae dp
* ScreenplayTextNo-Indent sptx1
* ScreenplayText sptx
* ScreenplayExtract sp
* SidebarHead-Level-1sbh1
* SidebarHead-Level-2sbh2
* SidebarTextsbtx
* SidebarTextNo-Indentsbtx1
* SidebarListBulletsbbl

Also add support for these new elements:

* aside

### PR DESCRIPTION
**Pull request by nelliemckesson**
I added the new styles as separate scss files, which is going to be the new thing going forward (every style gets it's own scss). @mattretzer can you review?

### CODE REVIEW

File: pdfmaker/css/_modules/_SidebarHead-Level-2sbh2.scss [hunk 1]
```diff
@@ -0,0 +1,18 @@
+.SidebarHead-Level-2sbh2  {
+  page-break-after: avoid;
+  font-size: $sizeextracthead;
+  line-height: $gridheight;
+  font-weight: bold;
+  fon-style: italic;
```
Comment (mattretzer) [inline pdfmaker/css/_modules/_SidebarHead-Level-2sbh2.scss]: Line 6 here should be font-style, correct?

File: pdfmaker/css/_modules/_Head-Level-1h1 [hunk 2]
```diff
@@ -0,0 +1,5 @@
+_Head-Level-1h1
```
Comment (mattretzer) [inline pdfmaker/css/_modules/_Head-Level-1h1]: I may just be a CSS newbie but- Is line 1 here an orphaned line?
& is it intentional for there to be no font-size specified in line 4?

File: pdfmaker/css/_modules/_SidebarHead-Level-2sbh2.scss (hunk 1 above)
Comment (nelliemckesson) [inline pdfmaker/css/_modules/_SidebarHead-Level-2sbh2.scss]: At last! You got one!

File: pdfmaker/css/_modules/_Head-Level-1h1 (hunk 2 above)
Comment (nelliemckesson) [inline pdfmaker/css/_modules/_Head-Level-1h1]: Oops. I didn't mean to include that file in the push! I'll remove it.

**Comment by nelliemckesson** [comment]
ok @mattretzer ready for another review

**Pull request merged**